src/api/app/assets/stylesheets/webui/application/dialog.scss

```diff
@@ -40,6 +40,17 @@
       font-weight: bold;
       font-size: 1.2em;
     }
+    .close_dialog {
```

Comment:
Why not applying the dialog styling via a generic class? That way it can be re-used it elsewhere.

And that's independent from how js get's applied. So there you could still identify elements via their id.